backend/code_review_backend/issues/api_v2.py

```diff
@@ -0,0 +1,121 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Please create a dedicated module for v2 api as `code_review_backend.issues.v2` where you'll have:
- `api.py` with the new endpoint + urls declaration
- `serializers`
- `tests` folder